vendor/github.com/gorilla/rpc/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2012 Rodrigo Moraes. All rights reserved.
```

Comment:
My point is that maybe we should just implement our own encode/decode rather than importing ~500 lines of code we're barely going to use...